### PR TITLE
improvement: enable custom duration and simplify expiration calculation

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed 
Allows setting a custom ban duration and simplifies the expiration time calculation.

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by omarcopires** [comment]
The behavior of the `/ban` and `/ipban` commands has been modified to allow specifying a custom number of days instead of using the fixed value of 7 days. Additionally, the code's readability has been improved by using `string.format` for the return messages.

All TalkActions have been centralized into a single file (revscript), making the code easier to maintain, organize, and access.
A new function has also been added to the Game to simplify access to a player's accountId.
Although I have not tested all the changes, I am available to address any issues or make further adjustments if needed.

**Comment by omarcopires** [comment]
The behavior of the `/ban` and `/ipban` commands has been modified to allow specifying a custom number of days instead of using the fixed value of 7 days. Additionally, the code's readability has been improved by using `string.format` for the return messages.

All TalkActions have been centralized into a single file (revscript), making the code easier to maintain, organize, and access.
A new function has also been added to the Game to simplify access to a player's accountId.
Although I have not tested all the changes, I am available to address any issues or make further adjustments if needed.

File: data/scripts/talkactions/ban.lua [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+	local resultId = db.storeQuery("SELECT 1 FROM `account_bans` WHERE `account_id` = " .. accountId)
+	if resultId then
+		result.free(resultId)
+		return true
+	end
+
+	local currentTime = os.time()
+	local expirationTime = currentTime + (banDuration * 24 * 60 * 60)
+	db.query(string.format("INSERT INTO `account_bans` (`account_id`, `reason`, `banned_at`, `expires_at`, `banned_by`) VALUES (%d, %s, %d, %d, %d)", accountId, db.escapeString(banReason), currentTime, expirationTime, player:getGuid()))
```
Comment (EPuncker) [inline data/scripts/talkactions/ban.lua:36]: line too long, break it as it used to be

File: data/scripts/talkactions/ban.lua [hunk 2]
```diff
@@ -0,0 +1,133 @@
+local ban = TalkAction("/ban")
+
+function ban.onSay(player, words, param)
+	if not player:getGroup():getAccess() then
```
Comment (EPuncker) [inline data/scripts/talkactions/ban.lua:4]: remove this check and use

```ban:access(true)```

between the separator and register methods, same with the others

**Review by EPuncker — CHANGES_REQUESTED** [review]

**Review by EPuncker — APPROVED** [review]

**Pull request merged**